docs/rkt/README.md

```diff
@@ -0,0 +1,34 @@
+# Integrating the Calico Network Plugin with rkt
+
+This guide will describe the configuration required to use the Calico network plugin in your rkt deployment.
+
+## Using rkt Plugins
+
+The CoreOS [documentation](https://github.com/coreos/rkt/blob/master/Documentation/networking.md) for Network Plugins will walk you through the basics of setting up networking in rkt.
+
+## Requirements
+
+* A working [etcd](https://github.com/coreos/etcd) service
+* A build of `calicoctl` after [projectcalico/calico-docker@10460cc405](https://github.com/projectcalico/calico-docker/commit/10460cc405f5aa4bc9ccb1fcaf8760088ae1ebf9)
```

Comment:
We can use a release version of calicoctl here - we don't need to point to a commit.